src/interfaces/coral_web/src/hooks/experiementalFeatures.ts

```diff
@@ -0,0 +1,21 @@
+import { useQuery } from '@tanstack/react-query';
+
+import { useCohereClient } from '@/cohere-client';
+
+export const useExperimentalFeatures = () => {
+  const cohereClient = useCohereClient();
+
+  return useQuery<boolean>({
+    queryKey: ['experimentalFeatures'],
+    queryFn: async () => {
+      try {
+        const data = await cohereClient.listExperimentalFeatures();
+        return data.USE_EXPERIMENTAL_LANGCHAIN;
```

Comment:
Feel like this is fine for now since we just have 1 experimental feature? If more of them get added, we can modify what we return based on design requirements

Reply:
I think I'd prefer to return the entire object so that it's clearer whenever we're using the hook and also we don't need to update this in the future when we do have other flags. but if we feel strongly about this we should rename the hook to `useIsExperimentalLangchainOn`

Reply:
opting to return entire object